camera/camera_manager.hh

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <yaml-cpp/yaml.h>
+#include <sys/types.h>
+#include <dirent.h>
+#include <iostream>
+#include <string>
+
+//%deps(opencv)
+#include <opencv2/opencv.hpp>
+
+namespace jet {
+
+struct Calibration {
+    cv::Mat camera_matrix;
+    cv::Mat distortion_coefficients;
+};
+struct Camera {
+    std::string serial_number = "CA38DB5E";
```

Comment:
Don't give it a default value -- this way, we can use `valgrind` to detect if it's been used without being set. 

An "uninitialized" flag is also reasonable. One should prefer to not have partially valid default behavior, and instead totally fail unless fully configured.